gnmvidispine/vs_metadata.py

```diff
@@ -55,12 +55,12 @@ def toXML(self,mdGroup=None):
                 if isinstance(line,datetime):
                     line = line.strftime("%Y-%m-%dT%H:%M:%S%Z")
 
-                valueEl.text = unicode(line)
+                valueEl.text = str(line)
```

Comment:
again, i think using str() here is dangerous as if `line` is a bytestring it will get a `b'` prepended. it's probably best to use a helper function something like:

```
def always_string(maybe_string):
    if isinstance(maybe_string,str):
        return maybe_string
    elif isinstance(maybe_string, bytestring):
        return maybe_string.decode("utf8")
    else:
        raise TypeError("always_string got an unexpected type {0}".format(type(maybe_string))
```

and use it in place of str(). That way we'll always get a utf8 string or an error if it wasn't a string type.

Reply:
Done.